### ISSUE TITLE
does vit-pytorch has mvit?

### ISSUE THREAD

**Issue by RyanHuangNLP** [lucidrains/vit-pytorch#190]
mvit is very good at two paper [Improved Multiscale Vision Transformers for Classification and Detection](https://arxiv.org/pdf/2112.01526v1.pdf), [Masked Feature Prediction for Self-Supervised Visual Pre-Training](https://arxiv.org/pdf/2112.09133v1.pdf), does vit-pytorch has it?

**Comment by lucidrains** [comment]
@RyanHuangNLP hmm, no it doesn't - are you waiting for them to release the code? the only novel thing in the paper is the residual pooling connection
